gcd/build.gradle

```diff
@@ -5,8 +5,9 @@ repositories {
 
 dependencies {
 
-    // TODO:2015-11-19:alexander.litus: change local dependency to Maven central repository dependency
+    // TODO:2015-11-19:alexander.litus: change local dependencies to Maven central repository dependency
     compile group: 'org.spine3', name: 'core-java', version: '0.1', changing: true, classifier: 'test'
```

Comment:
Because this is the dependency on the test classes (we need to extend them and use in tests).